public/themes/pterodactyl/js/frontend/console.js

```diff
@@ -149,10 +151,13 @@ function pushToTerminal(string) {
 
     if (TerminalQueue.length > 0) {
         for (var i = 0; i < CONSOLE_PUSH_COUNT && TerminalQueue.length > 0; i++) {
+            const scrolledDown = isTerminalScrolledDown();
```

Comment:
This should't be a `const` since this isn't a file that is transpired into ES5. In addition, why not just call `if (isTerminalScrolledDown()) below for scrolling down. Its a bit inconsistent here.

Reply:
My thought probably was that after a line is added the terminal isn't scrolled down anymore. I'll also move that part outside the loop.